packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -43,4 +48,15 @@ App::App() noexcept {
   InitializeComponent();
 }
 
+/// <summary>
+/// Invoked when the application is launched normally by the end user.  Other entry points
+/// will be used such as when the application is launched to open a specific file.
+/// </summary>
+/// <param name="e">Details about the launch request and process.</param>
+void App::OnLaunched(LaunchActivatedEventArgs const &e) {
+  super::OnLaunched(e);
+  Frame rootFrame = Window::Current().Content().as<Frame>();
```

Comment:
Just observing: Aleksander's PR has `window.Activate();` before navigating...

Reply:
just seeing this now. It's not necessary to call Window::Activate _if_ you call super::OnLaunched. Otherwise you must call it.

Reply:
It works without it. Like Alexander says it is redundant in this context.

---
In reply to: [432967013](https://github.com/microsoft/react-native-windows/pull/5071#discussion_r432967013) [](ancestors = 432967013)